birdhouse/components/cowbird/config/cowbird/config.yml.template

```diff
@@ -107,7 +107,7 @@ sync_permissions:
       - "geoserver_workspace : createStoredQuery <-> thredds_workspace : write"
   weaver_outputs:
     services:
-      weaver:
+      api:
```

Comment:
This definition doesn't match the example https://github.com/Ouranosinc/cowbird/blob/master/config/config.example.yml
The example does not have the repeated
```yaml 
 - name: weaver
   type: service
```
for each resource to sync as defined below.

We can also see in https://github.com/Ouranosinc/cowbird/blob/master/config/config.example.yml#L58 that "service name" is used, which brings more confusion. It should be service types instead?

Is the example out of date?

Reply:
You're right. It should indeed be "service type", not "service name".
I will add a commit here and to my other PR to review this and make it more explicit that it is a service type.